src/map/skill.c

```diff
@@ -94,7 +95,8 @@ int skill_splash_target(struct block_list* bl)
 }
 
 /// Returns the id of the skill, or 0 if not found.
-int skill_name2id(const char* name) {
+int skill_name2id(const char* name)
```

Comment:
i'm not against your changes, but i really hate such declaration of pointers. This is really annoing and confuse me and not only me so many times... What reason for writing like this? Sure, it's not your fault, but i saw really many lines of code with such declaration...